tests/release/pipelines/push_to_external_registry.go

```diff
@@ -33,7 +34,25 @@ var _ = framework.ReleasePipelinesSuiteDescribe("[HACBS-1571]test-release-e2e-pu
 	var fw *framework.Framework
 	AfterEach(framework.ReportFailure(&fw))
 	var err error
-	var devNamespace, managedNamespace, compName, additionalCompName string
+	var devNamespace, managedNamespace string
+	var compName = "dc-metro-map"
```

Comment:
These values can be defined as constants since they are not expected to change during test case is running.

Reply:
Typically I'd agree with you, but we only have 2 pipeline tests so far. We will have more. I think it is quite likely others will use it